src/encord_active/lib/model_predictions/writer.py

```diff
@@ -44,16 +51,20 @@ class LabelEntry:
     url: str
     img_id: ImageIdentifier
     class_id: ClassID  # TODO remove this stupid legacy thing.
-    x1: float = 0.0
-    y1: float = 0.0
-    x2: float = 0.0
-    y2: float = 0.0
+    x1: Optional[float] = None
+    y1: Optional[float] = None
+    x2: Optional[float] = None
+    y2: Optional[float] = None
     rle: Optional[RLEData] = None
```

Comment:
Are these always present or absent together? Maybe having some sort of other structy thing to denote that could be nicer?

Reply:
Probably would, but that is a hole we should not dive into right now since these would have loads of implications. (we save these and load them to dataframes)